config/space-presets.json

```diff
@@ -48,7 +48,7 @@
     }
   },
   {
-    "id": "5apps",
+    "id": "xmpp-5apps",
```

Comment:
Sorry, I still don't understand what that means really. Why should that not work?